packages/commonwealth/server/util/cosmosProxy.ts

```diff
@@ -27,36 +28,78 @@ function setupCosmosProxy(
     bodyParser.text(),
     calcCosmosRPCCacheKeyDuration,
     cacheDecorator.cacheMiddleware(
-      defaultCacheDuration,
+      DEFAULT_CACHE_DURATION,
       lookupKeyDurationInReq,
     ),
     async function cosmosProxy(req, res) {
```

Comment:
Correct me if I am wrong, but I think we can avoid a lot of duplication with the cosmosLCD api. This is since the both only depend on making either RPC or REST calls, but the contents are otherwise the same. I suggest you have one setupCosmosProxy that takes in either `'rest' | 'rpx'` and then calls the correct queryProxy from there.

Reply:
Agree we don't need the cosmosLCD naming. There are some slight differences in the way we handle responses and caching. But I can DRY it up some more. Happy to just have the cosmosAPI route with a v1 modifier.

Reply:
Renamed to `cosmosAPI/v1`and DRYed up code more.